package.json

```diff
@@ -5,8 +5,12 @@
   "scripts": {
     "start": "gulp",
     "build": "gulp build",
-    "lint": "eslint .",
-    "precommit": "lint-staged"
+    "lint": "eslint ."
+  },
+  "husky": {
+	  "hooks": {
+      "pre-commit": "lint-staged"
+	  }
```

Comment:
I think there's something weird in the indentation of lines 12 an 13. Would you mind checking it, please?